### DISCUSSION TITLE
Save current layout to file and load layout

### DISCUSSION THREAD

**Discussion by lukesmurray** [kovidgoyal/kitty#3368] [Q&A] [answered]
I know kitty has `kitty @ ls` which is pretty cool but the `ls` output doesn't include the layout information. Specifically I would like to save my current tabs, tab titles, and split layout to a file and restore them on start up. I don't need to restore any running programs I just want to restore the titles, the layout, and the working directory. Does anyone know if saving and restoring a layout is possible? I use `splits` so if the solution only works for splits that is totally fine.

**Comment by kovidgoyal** [comment, marked as answer]
No, its not, there is an open issue for this.
